ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/DDLCommandExec.java

```diff
@@ -61,7 +61,7 @@ private static DDLCommandResult executeOnMetaStore(DDLCommand ddlCommand, MetaSt
       return ddlCommand.run(metaStore);
     } catch (Exception e) {
       String stackTrace = ExceptionUtil.stackTraceToString(e);
-      LOGGER.error(stackTrace);
+      LOGGER.error("executeOnMetaStore:" + ddlCommand + " failed", e);
```

Comment:
should this be warn? I see error as something that is going to blow everything up, but this continues.

Reply:
warn and string.format